Cookbook/Technical-Documents/AppCenter.md

```diff
@@ -0,0 +1,61 @@
+# Accessing App Center builds
+
+### Steps to install
+1. On your device, visit [App Center](http://appcenter.ms/apps)
+
+2. As a login method, select `Company or school`
+
+	![app-center-company-or-school](./Assets/app-center/company-or-school.png)
```

Comment:
The images are not loading, the reference is probably wrong, maybe `Assets/app-center/company-or-school.png`?

Reply:
They aren't working in the diff, you need to select “View file” from the dropdown menu for them to work :)

Reply:
:disappear: 😆